build/common.go

```diff
@@ -105,6 +115,7 @@ func newBuildConfig(os string, arch string) Config {
 		Arch:        arch,
 		EnableDebug: false,
 		Env:         map[string]string{},
+		EnableCGO:   true,
```

Comment:
I don't think we should enable cgo by default:
```suggestion
```

Reply:
correct! I've just reverted it